data/json/items/armor.json

```diff
@@ -966,7 +966,7 @@
     "coverage": 40,
     "warmth": 10,
     "material_thickness": 1,
-    "flags": [ "VARSIZE", "WATER_FRIENDLY", "FANCY" ]
+    "flags": [ "VARSIZE", "WATER_FRIENDLY", "FANCY", "HELMET_COMPAT" ]
```

Comment:
Berets are compatible with headsets (which is why tank units adopted them early) but not helmets as far as I can tell on further research.